test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -94,6 +95,76 @@ public void It_publishes_self_contained_apps_to_the_publish_folder_and_the_app_s
                 .HaveStdOutContaining("Hello World!");
         }
 
+        //Note: Pre Netcoreapp2.0 stanalone activation uses renamed dotnet.exe
+        //      While Post 2.0 we are shifting to using apphost.exe, so both publish needs to be validated
+        [Fact]
+        public void Publish_standalone_post_netcoreapp2_app_and_it_should_run()
+        {
+            var targetFramework = "netcoreapp2.0";
+            var rid = EnvironmentInfo.GetCompatibleRid(targetFramework);
+
+
+            TestProject testProject = new TestProject()
+            {
+                Name = "Hello",
+                IsSdkProject = true,
+                TargetFrameworks = targetFramework,
+                RuntimeFrameworkVersion = RepoInfo.NetCoreApp20Version,
+                RuntimeIdentifier = rid,
+                IsExe = true,
+            };
+            
+
+            testProject.SourceFiles["Program.cs"] = @"
+using System;
+public static class Program
+{
+    public static void Main()
+    {
+        Console.WriteLine(""Hello from a netcoreapp2.0.!"");
+    }
+}
+";
+            var referencerAsset = _testAssetsManager.CreateTestProject(testProject);
+
+            referencerAsset.Restore(testProject.Name);
+            var publishCommand = new PublishCommand(Stage0MSBuild, Path.Combine(referencerAsset.TestRoot, testProject.Name));
+            var publishResult = publishCommand.Execute($"/p:RuntimeIdentifier={rid}");
```

Comment:
You don't need to pass `/p:RuntimeIdentifier={rid}` to publish.  That value is already hard-coded into the .csproj up above.  You can just say `.Execute()`.